### ISSUE TITLE
Add folder to album

### ISSUE THREAD

**Issue by afonsofrancof** [nextcloud/photos#1365]
**Is your feature request related to a problem? Please describe.**

As nextcloud doesn't sort photos by creation date, my way of organizing them is by naming them with their creation date, so the photos default view doesnt work for me.
Also , I used to have folders to represent albums, and I would then share them with my friends.
There is currently no way to add photos to albums from the folders view (which is the only one that lets you sort by name), or even better, a whole folder to/as an album.
I think that would be a very good addition to the app.

**Describe the solution you'd like**
A way to add photos/folders to/as an album.

**Comment by skjnldsv** [comment]
Which nc version please ?

**Comment by afonsofrancof** [comment]
> Which nc version please ?

Nextcloud 25 RC5

**Comment by skjnldsv** [comment]
You can add any photos you want to a folder on 25 :)
![image](https://user-images.githubusercontent.com/14975046/196358560-4d716989-478c-4354-bdc0-28133b4ed4a4.png)

Now to understand your flow a little better, when you say "adding folders", you mean adding folders from your nextcloud or from your local computer?

**Comment by afonsofrancof** [comment]
> Now to understand your flow a little better, when you say "adding folders", you mean adding folders from your nextcloud or from your local computer?
> 
 
I mean adding folders from nextcloud to an album. On the photos app there is a folders tab , where you can browse your folders with photos,and it would be nice to be able to add photos to an album from there.

**Comment by skjnldsv** [comment]
Thanks for clarifying.
If you add photos to the said folder, would the album be automatically updated?

So you want to aggregate multiple photos and multiple folders into one album?

**Comment by afonsofrancof** [comment]
> Thanks for clarifying.
> 
> If you add photos to the said folder, would the album be automatically updated?
> 
No , I think that it would just add the photos on that folder (and subfolders) to the album. So just as a way to add them all in bulk.
> 
> So you want to aggregate multiple photos and multiple folders into one album?

Yes , I could then just go to the folders I want and add them to a specific album.

This would not have to change the current album implementation, it would just be another way to add photos to them. They would not be synced with the folder.

**Comment by nicokaiser** [comment]
Maybe this could be expanded to "allow multiple photos to be selected", e.g. via Shift key. Adding many photos from a folder to an album is currently very complicated, the photos need to be seleceted one by one. Selecting the first, and while holding Shift selecting the last, would be a good way to make this easier.

**Comment by skjnldsv** [comment]
> No , I think that it would just add the photos on that folder (and subfolders) to the album. So just as a way to add them all in bulk, but not link the folder dynamically to the album.

Awesome, thanks for clarifying :v

**Comment by afonsofrancof** [comment]
> Maybe this could be expanded to "allow multiple photos to be selected", e.g. via Shift key. Adding many photos from a folder to an album is currently very complicated, the photos need to be seleceted one by one. Selecting the first, and while holding Shift selecting the last, would be a good way to make this easier.

Yes , that would also be a good addition I think. There isn't even a way to add a single photo to an album from the folders view in the photos app.

**Comment by TheGroundZero** [comment]
Sort of bumping this one.

I added a SMB share as external storage to my Nextcloud (running on TrueNAS) and would now like to import the folder structure of my Photos share as albums in Nextcloud.

In my scenario, I wouldn't mind the album be linked to the folder, to ensure there're no unnecessary duplicates being created.
Having the option to at least select all images using shift+click to select ranges would already be somewhat helpful, but doing so for each and every picture folder on my network share, would still be cumbersome.

**Comment by JohannesBe** [comment]
If at all interested, I'm willing to put down a 100$ incentive out of my personal pocket to accelerate the implementation of folder-based albums (i.e. not a 1-time import, but rather a sustainable solution). Any takers?
